Error Linking rpcs3

Hi, log of error of the compilation:

[ 97%] Building CXX object rpcs3/CMakeFiles/rpcs3.dir/__/Utilities/rFile.cpp.o
[ 98%] Building CXX object rpcs3/CMakeFiles/rpcs3.dir/__/Utilities/SSemaphore.cpp.o
[ 98%] Building CXX object rpcs3/CMakeFiles/rpcs3.dir/__/Utilities/Log.cpp.o
[ 98%] Building CXX object rpcs3/CMakeFiles/rpcs3.dir/__/Utilities/rPlatform.cpp.o
[100%] Building CXX object rpcs3/CMakeFiles/rpcs3.dir/__/Utilities/AutoPause.cpp.o
[100%] Building CXX object rpcs3/CMakeFiles/rpcs3.dir/__/Utilities/rMsgBox.cpp.o
Linking CXX executable ../bin/rpcs3
/usr/lib/gcc/x86_64-pc-linux-gnu/4.8.3/../../../../x86_64-pc-linux-gnu/bin/ld: CMakeFiles/rpcs3.dir/Emu/Memory/vm.cpp.o: referencia sin definir al símbolo 'shm_unlink@@GLIBC_2.2.5'
/lib64/librt.so.1: error adding symbols: DSO missing from command line
collect2: error: ld devolvió el estado de salida 1
rpcs3/CMakeFiles/rpcs3.dir/build.make:7360: recipe for target 'bin/rpcs3' failed
make[2]: *** [bin/rpcs3] Error 1
CMakeFiles/Makefile2:263: recipe for target 'rpcs3/CMakeFiles/rpcs3.dir/all' failed
make[1]: *** [rpcs3/CMakeFiles/rpcs3.dir/all] Error 2
Makefile:116: recipe for target 'all' failed
make: *** [all] Error 2

--- a/rpcs3/CMakeLists.txt
+++ b/rpcs3/CMakeLists.txt
@@ -120,7 +120,7 @@ set_source_files_properties(${RPCS3_SRC_DIR}/Emu/Cell/PPULLVMRecompiler.cpp PROP
 add_executable(rpcs3 ${RPCS3_SRC})
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -L${CMAKE_CURRENT_BINARY_DIR}/../asmjit/") #hack because the asmjit cmake file force fno exceptions
-target_link_libraries(rpcs3  asmjit.a  ${wxWidgets_LIBRARIES} ${OPENAL_LIBRARY} ${GLEW_LIBRARY} ${OPENGL_LIBRARIES} libavformat.a libavcodec.a libavutil.a libswresample.a libswscale.a ${ZLIB_LIBRARIES} ${LLVM_LIBS})
+target_link_libraries(rpcs3  asmjit.a  ${wxWidgets_LIBRARIES} ${OPENAL_LIBRARY} ${GLEW_LIBRARY} ${OPENGL_LIBRARIES} libavformat.a libavcodec.a libavutil.a libswresample.a libswscale.a ${ZLIB_LIBRARIES} ${LLVM_LIBS} rt)
 
 set_target_properties(rpcs3 PROPERTIES COTIRE_CXX_PREFIX_HEADER_INIT "${RPCS3_SRC_DIR}/stdafx.h")
 cotire(rpcs3)

